### ISSUE TITLE
Configuration with no Debug schemes causes a failure in react-native-xcode.sh 

### ISSUE THREAD

**Issue by gwmccull** [Thinkmill/react-native-schemes-manager#7]
### Steps to reproduce the behavior
add a package.json with:
```json
{
  "xcodeScheme: {
    "Release": [ "Beta" ]
  }
}

will generate the following in Xcode:
```bash
export NODE_BINARY=node
export DEVELOPMENT_BUILD_CONFIGURATIONS="|Debug"
../node_modules/react-native-schemes-manager/lib/react-native-xcode.sh
```
The leading bar `|` causes an error to be thrown by `react-native-xcode.sh`

### Expected behavior
the export shouldn't have the leading bar `|`

**Comment by gwmccull** [comment]
fixed in 
https://github.com/Thinkmill/react-native-schemes-manager/pull/6/files

**Comment by blargity** [comment]
Ah, yeah, that'd do it. Thanks for the fix!

**Comment by gwmccull** [comment]
Fixed in beta 11 #9
